UI/window-basic-main-transitions.cpp

```diff
@@ -1865,3 +1868,27 @@ int OBSBasic::GetOverrideTransitionDuration(OBSSource source)
 
 	return (int)obs_data_get_int(data, "transition_duration");
 }
+
+void OBSBasic::UpdateStudioModeSceneLabels()
+{
+	if (!IsPreviewProgramMode())
+		return;
+
+	QString program = QTStr("StudioMode.Program");
```

Comment:
Would this function be cleaner with QString `.arg()`?

Reply:
We would need to add a new string to the en-US.ini file, so I feel like this is just fine.

Reply:
Not really. `QString("%1 - %2").arg(program, programSceneName)`

Reply:
Updated to use the args.